src/components/CreateNewBook.js

```diff
@@ -40,19 +39,13 @@ const CreateNewBook = () => {
             placeholder="Book title"
           />
           <Form.Control
-            value={authorInput}
-            onInput={(e) => setAuthorInput(e.target.value)}
+            value={categoryInput}
+            onInput={(e) => setCategoryInput(e.target.value)}
             type="text"
             required
-            placeholder="Book Author"
+            placeholder="Category"
           />
         </Form.Group>
-        <Form.Select required aria-label="Default select example">
-          <option>Category</option>
-          <option value="1">Adventure</option>
-          <option value="2">Classics</option>
-          <option value="3">Horror</option>
-        </Form.Select>
         <Button variant="primary" type="submit" onClick={handleSubmit}>
           Add
         </Button>
```

Comment:
- [ ] Optional suggestion

 When a user adds or sends information, the input has to be empty so that a user can add other information.
 
![user-14](https://user-images.githubusercontent.com/55635977/140184859-37680ce3-5649-45d6-9038-19e6ece519f6.png)

Reply:
I will add this feature, it's a great thing! Thank's @Emmyn5600 for this and for the review! 😄